src/use_case/ranking/RankingInteractor.java

```diff
@@ -1,3 +1,70 @@
+package use_case.ranking;
+
+import entity.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class RankingInteractor implements RankingInputBoundary {
+    private RankingOutputBoundary presenter;
+
+    public RankingInteractor(RankingOutputBoundary presenter) {
+        this.presenter = presenter;
+    }
+
+    @Override
+    public void execute(RankingInputData inputData) {
```

Comment:
You should write Javadocs here. I would prefer to know what exactly the interactor does, at least on a high level.